### PR TITLE
Storybook: add viewport addon

### LINKED ISSUE #531: Storybook viewport add on
**Is your feature request related to a problem? Please describe.**
UX have requested if we can add the `viewport` storybook add on to our implementation. 
We should evaluate if the add on appears to be useful and easy to use with our implementation.

https://github.com/storybooks/storybook/tree/master/addons/viewport

**Describe the solution you'd like**
Install and use the `viewport` add on in psammead storybook. 

**Describe alternatives you've considered**
Discuss with UX why we don't think we should use the add on/suggest a different add on.

**Testing notes**
[Tester to complete]

Dev insight: Regression of storybook

**Additional context**
Laura and Fan are the UX members who have requested this.

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves #531 

**Overall change:** 
Add viewport addon to Storybook

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~~Tests added for new features~~
- [x] Test engineer approval

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-| 1.1.6   | [PR#514](https://github.com/BBC/psammead/pull/514) Upgrade to Node 10 
+| 1.1.6   | [PR#TBA](https://github.com/BBC/psammead/pull/TBA) Add viewport addon |
```
Comment (DenisHdz) [inline CHANGELOG.md]: 🙈

```suggestion
| 1.1.7   | [PR#536](https://github.com/BBC/psammead/pull/536) Add viewport addon |
```

Comment (j-pendlebury) [reply CHANGELOG.md]: Thanks Denis! 😄

File: .storybook/config.js [hunk 2]
```diff
@@ -76,6 +76,7 @@ export const ampDecorator = story => (
 
 function loadAllStories() {
   require('glob-loader!./stories.pattern');
+  addParameters({ viewport: options });
```
Comment (ghost) [inline .storybook/config.js]: Undefined `options`
Screenshot 2019-05-20 at 13 56 05

**Review by ghost — COMMENTED** [review]
Undefined `options`

File: .storybook/config.js (hunk 2 above)
Comment (ghost) [inline .storybook/config.js]: We also don't need to call `addParameters` the addon gets loaded automatically.

Comment (j-pendlebury) [reply .storybook/config.js]: Thanks Isaac! Fixed now 😄

**Review by ghost — APPROVED** [review]
Looks Great.

**Review by DenisHdz — APPROVED** [review]

**Comment by j-pendlebury** [comment]
image
^ viewport addon screenshot

**Comment by david-boydell** [comment]
Looks good, had a look with Laura and UX seems happy thanks.

**Pull request merged**